src/styles/main.scss

```diff
@@ -0,0 +1,95 @@
+// extends
+%arrowsPosition {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+
+  transform-origin: left center;
+}
+
+// variables
+$minuteColor: #f00;
+$secondColor: #00f;
+$clockRimColor: #000;
+
+$secondArrowLength: 500px;
+$secondArrowWidth: $secondArrowLength / 25;
+
+$minuteArrowLength: $secondArrowLength / 2;
+$minuteArrowWidth: $minuteArrowLength / 10;
+
+$clockSize: $secondArrowLength * 2.5;
+$centerSize: $secondArrowLength / 10;
+
+// mixins
+@mixin circle($size) {
+  width: $size;
+  height: $size;
+
+  border-radius: 50%;
+}
+
+body {
+  margin: 0;
+}
+
+.main-content {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  height: 100vh;
+}
+
+.stop-watch {
+  position: relative;
+
+  @include circle($clockSize);
+
+  border: 5px solid $clockRimColor;
+
+  &__center {
+    @extend %arrowsPosition;
+    z-index: 1;
+
+    @include circle($centerSize);
+
+    transform: translate(-50%, -50%);
+
+    background-color: $secondColor;
+  }
+
+  &__second {
+    @extend %arrowsPosition;
+
+    width: $secondArrowLength;
+    height: $secondArrowWidth;
+
+    animation: clock 60s steps(60) infinite;
+
+    background-color: $secondColor;
+    border-radius: $secondArrowWidth;
+  }
+
+  &__minute {
+    @extend %arrowsPosition;
+
+    height: $minuteArrowWidth;
+    width: $minuteArrowLength;
+
+    animation: clock 60s steps(60) infinite;
```

Comment:
your clock doesn't work properly, check it for yourself& a minute clock should make one rotation in 1 hour, not a minute^ and also it should move smoothly